shell/platform/android/io/flutter/plugin/platform/VirtualDisplayController.java

```diff
@@ -23,6 +23,15 @@
 class VirtualDisplayController {
   private static String TAG = "VirtualDisplayController";
 
+  private static VirtualDisplay.Callback callback =
+      new VirtualDisplay.Callback() {
```

Comment:
question for my learning: Is using this callback with `onPaused()` and `onResumed()` overridden to do nothing somehow different from passing null as the callback to `createVirtualDisplay`?  Presumably `onStopped` will also do nothing without an explicit definition?

Reply:
The callback is currently a no-op but I wanted to leave the structure in the code for future debugging.

Interesting is that I've *never* had one of these methods be invoked and I've found some Stackoverflow posts asking why that is the case 🤷